src/utils/expression-runner.ts

```diff
@@ -67,6 +67,24 @@ export function evaluateExpression(
     return null;
   }
 
+  function calcBMI(heightQuestionId, weightQuestionId) {
```

Comment:
This question isn't related to this function, just wanted to ask if this the same feature that will be used to calculate other computed values like the next drug/appointment date or is this something that will be handled on the backend?

Reply:
I guess other advanced computations would required pre-defined helpers.